apps/dolly-frontend/src/main/js/src/components/ui/button/SlettButton/SlettButton.js

```diff
@@ -34,7 +36,8 @@ export const SlettButton = ({ action, loading, children, disabled = false }) =>
 						<NavButton
 							onClick={() => {
 								closeModal()
-								return action()
+								action(gruppeId)
```

Comment:
SlettButton blir også brukt for å denne for å slette maler, som ikke funker nå. Må enten fjerne gruppeId delen eller lage egen button for maler.

Reply:
Prøvde også å slette en ident, men ble da redirected til gruppeoversikten etter sletting.